### DISCUSSION TITLE
What does LmHandlerSetSystemMaxRxError do?

### DISCUSSION THREAD

**Discussion by AD-NCD** [Lora-net/LoRaMac-node#1552] [Q&A]
Hello, i was wondering how does LmHandlerSetSystemMaxRxError works? does it open window before it's starting time or after or does it shrink it from the middle? so if i set it to 100 msec what will happen? i check documentation and i couldn't find anything.
Thanks

**Comment by gopalsih** [comment]
Can someone please answer this , i have the same question.

**Comment by mluis1** [comment]
Please refer to
https://github.com/Lora-net/LoRaMac-node/blob/dcbcfb329b4a343ab007bc19ac43a8dc952b3354/src/mac/region/Region.h#L940-L993
